gcloud/bigquery/_helpers.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""BigQuery utility functions."""
+
+
+import datetime
+import sys
+
+import pytz
+
+_EPOCH = datetime.datetime(1970, 1, 1, tzinfo=pytz.utc)
```

Comment:
I want to ensure that the epoch is a non-naive datetime (forcing it to use `pytz.utc` as its zone), so I went with the explicit constructor rather than one of the classmethods.